<launch>
    <arg name="mode"/>
    <node pkg="hrl_rfh_summer_2011" name="sm_touch_face" type="sm_touch_face.py" output="screen"
          args="$(arg mode)">
        <param name="arm" type="str" value="r"/>
        <param name="tool_frame" type="str" value="r_scratcher_touch"/>
        <param name="tool_approach_frame" type="str" value="r_scratcher_approach"/>
        <param name="base_offset_frame" type="str" value="optimal_head_location"/>
        <param name="nav_approach_dist" type="double" value="0.3"/>
        <remap from="/pixel3d" to="/wt_pixel3d"/>
    </node>
    <node pkg="hrl_rfh_summer_2011" name="sm_touch_face_diagnostics" type="sm_touch_face_diagnostics.py" output="screen"/>
</launch>
